src/pages/style-guide.mdx

```diff
@@ -0,0 +1,162 @@
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-dark)"}}></div>
+                <div class="color-text">#2f7a1e</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-darker)"}}></div>
+                <div class="color-text">#2c731c</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-darkest)"}}></div>
+                <div class="color-text ">#245e17</div>
+            </div>
+        </div>
+    </div>
+</div>
+
+### Dark theme
+
+<div class="color-palette-dark-theme-container">
+    <div class="row margin-bottom--md">
+        <div class="col col--3">
+            <span class="text--bold">Primary color</span>
+        </div>
+        <div class="col">
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary)"}}></div>
+                <div class="color-text">#43b02a</div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col col--3">
+            <span class="text--bold">Color variations (light to dark)</span>
+        </div>
+        <div class="col">
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-lightest)"}}></div>
+                <div class="color-text">#62d348</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-lighter)"}}></div>
+                <div class="color-text">#4dca30</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-light)"}}></div>
+                <div class="color-text">#4ac22e</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary)"}}></div>
+                <div class="color-text">#43b02a</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-dark)"}}></div>
+                <div class="color-text">#3c9e26</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-darker)"}}></div>
+                <div class="color-text">#399624</div>
+            </div>
+            <div class="color-container">
+                <div class="color" style={{"background-color": "var(--ifm-color-primary-darkest)"}}></div>
+                <div class="color-text">#2f7b1d</div>
```

Comment:
This doesn't pass WCAG AA(A) [contrast-checker](https://webaim.org/resources/contrastchecker/):

<img width="917" alt="image" src="https://user-images.githubusercontent.com/2559438/226867914-91ffd28a-fbb4-407b-a181-df5a6b99e033.png">

Reply:
@jaimergp, that color shouldn't be used for text. Only the primary color should be used for text, and links too.

Should that be stated somewhere? 🤷‍♂️

Reply:
Are there colors that are permitted to use as text? Like if someone wanted to emphasize bolding with a green. (Also, the hyperlink standards Jaime referenced above. Could the hover be one of these greens?)

Reply:
I think specifying whether colors are okay for text is a slippery slope because ultimately it's the appropriate contrast that will make it work or not.

What I will add to the style guide is that we have approved the primary color and background color contrast because it passes the WCAG.